### PR TITLE
Update check-dependencies to read new format of `yarn outdated`

### PR DESCRIPTION
**Pull request by jrwats**
The field for `type` is 5 when workspaces are specified in the project. Let's just use the indices specified in `head`.

### CODE REVIEW

**Comment by kassens** [comment]
Can you rebase on #333? That changed the CI config as well, I think with the same change.

**Comment by jrwats** [comment]
Wow, maybe I'm dumb and it's actually node's version that matters?  I'm running v10.11.0 where my fix is "necessary".

File: packages/fbjs-scripts/gulp/check-dependencies.js [hunk 1]
```diff
@@ -38,17 +38,27 @@ module.exports = function(opts) {
...
-          .map(d => JSON.parse(d))[1]['data']['body'];
+          .map(d => JSON.parse(d))
+          .filter(j => j.type === 'table')[0].data;
       } catch (e) {
         console.log('error', e)
         cb(new PluginError(PLUGIN_NAME, 'npm broke'));
       }
 
+      // Convert ["Package", "Current",...] to {"Package": 0, ...}
+      const name2idx = {};
```
Comment (zpao) [inline packages/fbjs-scripts/gulp/check-dependencies.js]: > `name2idx`

ಠ_ಠ

Comment (jrwats) [reply packages/fbjs-scripts/gulp/check-dependencies.js]: nameToIdx?

**Comment by zpao** [comment]
Alright, rebased and squashed into a single commit: https://github.com/facebook/fbjs/commit/ccad8f1e81daa851d18cc84153b5e1d595f1edef

**Pull request closed without merging**